src/Containers-OrderPreservingDictionary/CTStandardOrderedDictionary.class.st

```diff
@@ -7,6 +7,23 @@ For example, the printing is using the insertion order
 
 	CTStandardOrderedDictionary orderedDictionaryABC printString 
 	>>> 'a CTStandardOrderedDictionary(#A->''A'' #B->''B'' #C->''C'')'
+
+I work mainly as a Dictionary except that I also store keys in an Array to keep track of the order of elements.
+	
+If I try to access key that doesn't exist an error is raised.
```

Comment:
Maybe the difference is that one has a default value and this one raises an error?
Wouldn't it be good to make that a strategy? (like, if we set a default value we use it, or we can say `beStrict` to raise an error)